test/performance/odp_bench_buffer.c

```diff
@@ -681,16 +681,16 @@ static int bench_buffer_export(void *data)
 	int ret = 0;
 
 	if (test_common_write("%s", gbl_args->appl.time ?
-			      "Function name,Average nsec per function call\n" :
-			      "Function name,Average CPU cycles per function call\n")) {
+		"function name,average nsec per function call\n" :
+		"function name,average cpu cycles per function call\n")) {
```

Comment:
Function parameters ought to be aligned to the same level as the opening brace whenever possible for better readability. Same comment for all `test_common_write() calls`.